api/src/pcapi/core/users/api.py

```diff
@@ -1368,3 +1373,87 @@ def notify_users_before_deletion_of_suspended_account() -> None:
                 "Could not send email before deletion of suspended account",
                 extra={"user": account.id},
             )
+
+
+def anonymise_user(user: users_models.User, *, force: bool = False) -> None:
+    """
+    Anonymise the given User. If force is True, the function will anonymise the user even if it has an adress and we
+    cannot find an iris for it.
+    """
```

Comment:
```suggestion
def anonymize_user(user: users_models.User, *, force: bool = False) -> None:
    """
    Anonymize the given User. If force is True, the function will anonymize the user even if he has an adress and we
    cannot find an iris for it.
    """
```
et à plein d'autres endroits

Reply:
`Anonymise` est une orthographe acceptée, et par ailleurs l'orthographe anglaise (et pas américaine), si je ne m'abuse.
Et je suis partisan de l'anglais de nos voisins outre Manche

Reply:
J'ai deja tout armonisé sur Anonymise, pas envie de repasser a la version américaine.